src/configs/env.ts

```diff
@@ -199,6 +199,14 @@ export const SUPERSET_IRS_LITE_REPORTING_INDICATOR_ROWS =
   process.env.REACT_APP_SUPERSET_IRS_LITE_REPORTING_INDICATOR_ROWS || 'zambia2020';
 export type SUPERSET_IRS_LITE_REPORTING_INDICATOR_ROWS = typeof SUPERSET_IRS_LITE_REPORTING_INDICATOR_ROWS;
 
+export const SUPERSET_MDA_LITE_REPORTING_INDICATOR_ROWS =
+  process.env.REACT_APP_SUPERSET_MDA_LITE_REPORTING_INDICATOR_ROWS || 'zambia2020';
+export type SUPERSET_MDA_LITE_REPORTING_INDICATOR_ROWS = typeof SUPERSET_MDA_LITE_REPORTING_INDICATOR_ROWS;
+
+export const SUPERSET_MDA_LITE_REPORTING_INDICATOR_STOPS =
+  process.env.REACT_APP_SUPERSET_MDA_LITE_REPORTING_INDICATOR_STOPS || 'zambia2020';
+export type SUPERSET_MDA_LITE_REPORTING_INDICATOR_STOPS = typeof SUPERSET_MDA_LITE_REPORTING_INDICATOR_STOPS;
+
```

Comment:
Should the default value here be `zambia2020`?
Note: MDA-Lite plans are for kenya.

Reply:
@ciremusyoka 
followed IRS lite implementation, Let me know if this needs to be changed for MDA lite.

Reply:
The reason why `zambia2020` was used is because IRS was used for Zambia.